_data/m2.yml

```diff
@@ -43,4 +43,8 @@ solution_validation:
 
 conclusions:
     section: m2_conclusions
-    title: Conclusions
\ No newline at end of file
+    title: Conclusions
+    intro: From all the answers we gathered, we found that people rely on the internet when learning something new. Usually, they start from general aspects of those subjects, and after that, they go to particular information. The most popular resources are video tutorials, online articles and courses. But before selecting what materials to learn from, they have to know if they are trustworthy. To find that, they search reviews as notes or comments or ask acquaintances.
+    pr_val: The problems people often have when trying to study is that there is a lot of information on the internet and they do not know where to start. The information is not structured on their needs, it is sometimes incomplete or taken out of the context. These facts confirm the problem discovered by us. 
```

Comment:
Here instead of "not structured on their needs" I would say "not structured according to their needs"

Reply:
Cred ca suna mai bine "they move to (more) specific information."